env/release.mk

```diff
@@ -12,7 +12,7 @@ export SCP_OPTS=-P 22
 
 # where to put binary files
 export WARDIR=/packages/binary/war${RELEASELINE}
-export MSIDIR=C:\packages\binary\windows${RELEASELINE}
+export MSIDIR=/packages/binary/windows${RELEASELINE}
```

Comment:
no longer on c:/ ?

Reply:
It appears that powershell is doing the translatation, at least it's working :p.